.ci_support/migrations/cuda118.yaml

```diff
@@ -2,6 +2,7 @@ migrator_ts: 1692828152
 __migrator:
   kind:
     version
+  use_local: true # TODO: Remove this line after completing testing
```

Comment:
Not sure that we want to have the cross-compiled jobs for merging. We don't _need_ them from the POV of the CI setup itself, just for testing the branches here in CI. I suggested adding them in https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/236, but never got a response.

Reply:
Yep no strong feelings here. Happy to keep or drop as preferred